backend/src/Repository/PlantRepository.php

```diff
@@ -30,6 +30,29 @@ public function deletePlant(Plant $plant): void {
         $this->_em->flush();
     }
 
+    public function setTimeLeft(Plant $plant): Plant {
```

Comment:
please create a service class for that

no database access => not into the repository